src/App.js

```diff
@@ -66,7 +66,7 @@ const App = (): React$Element<typeof React.Fragment> => {
       <CssBaseline />
       <NavigationMenu />
       <Switch>
-        <Route path="/newdraft">
+        <Route path="/newdraft/:step">
```

Comment:
I think just having the step might not be enough. is it possible to make it relative to also the folder id ?

Reply:
So first folderID then step in the route maybe. But where is folderID used here in wizard creating submission? And how to handle non existing folderID, when the first two steps -1 and 0 are in question, keeping folderID -1? 
folderID is kept in global state, submissionFolderSlice. And in route /home/drafts or /home/published .
Maybe not in this PR?

Reply:
i assume the non-existing folderID would only be the case when `/newdraft` is first accessed and when a request is made to the folder API that is when we know it and we can use it in the URL (and included it with the previous and next links).

i was asking if possible, sure part of another PR is ok, I was thinking of what that would imply and if the changes in this PR would need to be redone. As such a feature might be necessary when we want to link folders from `/home/drafts` to be edited in the wizard. 

Does not have to be the only solution, however evaluating our options.

Reply:
One option with this task is to do it, is to keep steps in Redux state and keep route clean. So no steps in route and try to handle browsers history other wise.

Reply:
Id's suggest on navigating with query parameters. Example URL with params:
`/newdraft?step=1&folderid=123123`

Reply:
Also, check and handler for `folderId` in query params would fix current problem where user refreshes the browser.